Update single post template

mikachan opened the pull request:
<!-- You can erase any parts of this template not applicable to your Pull Request. -->

### Changes proposed in this Pull Request:

<!-- If necessary, indicate if this PR is part of a bigger feature. Add a label with the format `focus: name of the feature [team:name of the team]`. -->

<!-- Describe the changes made to this Pull Request and the reason for such changes. -->

Closes WOOTHME-80.

This updates the Single Post Template to match the new Figma design.

<img width="1509" alt="image" src="https://github.com/user-attachments/assets/c424c719-be1b-4fbe-9988-6bfad41f3e8a" />


<img width="1509" alt="image" src="https://github.com/user-attachments/assets/84c08d96-dd66-4771-b4e9-313195005013" />

mikachan committed:
Update single post template

mikachan committed:
Update post term styles

MaggieCabrera commented:
I think you might be looking at the wrong figma file? the cover block looks different to me:

<img width="946" alt="Screenshot 2025-06-20 at 17 58 56" src="https://github.com/user-attachments/assets/7f2bdc19-b92a-4ea6-8330-995ae288c54b" />

mikachan committed:
Update featured image alignment

mikachan commented:
I think I just missed the alignment of the image! Updated in https://github.com/woocommerce/woo-themes/pull/110/commits/1bb1d743575f7f54a330482781654f0d38473a3e. The Figma I'm looking at is Gum9XNWccPKfkVRFk1Whsp.

mikachan committed:
Reduce spacer height under image

MaggieCabrera reviewed (commented):
I think we are missing top spacing here:

<img width="1483" alt="Screenshot 2025-06-20 at 18 17 53" src="https://github.com/user-attachments/assets/5675cfa0-c63a-467e-9b7a-c636282f8ac8" />
<img width="946" alt="Screenshot 2025-06-20 at 18 17 41" src="https://github.com/user-attachments/assets/d850b716-4ceb-4230-ba24-544798a21201" />

If you don't have access to dev mode please request it and ping ellen or Bea if you don't get it quick, it makes life way easier

MaggieCabrera reviewed (commented):
The categories seem to have a button kind of style: 

<img width="1478" alt="Screenshot 2025-06-20 at 18 19 52" src="https://github.com/user-attachments/assets/dee07982-4836-4b11-8aa9-96fe14a97be9" />

mikachan commented:
Yeah I don't have dev mode, I'll request it.

> The categories seem to have a button kind of style:

These have been updated in theme.json but it doesn't come through in the Editor, I'll open an upstream issue for this.

MaggieCabrera reviewed (commented):
We are also missing this section, which should probably be a pattern (there is a very similar one in the home page):

<img width="953" alt="Screenshot 2025-06-20 at 18 21 15" src="https://github.com/user-attachments/assets/98ae6996-4b5c-4dcc-bb56-de1d3a221391" />

But we can't achieve "related posts" with core, which is unfortunate. Query loop should have an option called "related" if we are inserting the query loop in the single post template

mikachan committed:
Increase top spacing

MaggieCabrera commented:
Related GB issue https://github.com/WordPress/gutenberg/issues/55717

mikachan committed:
Use spacers for spacing

mikachan committed:
Update spacing to match Figma

mikachan committed:
Add More from the journal pattern

mikachan commented:
I've updated the spacing now based on Figma - now that I can see the spacing sizes 😅 

> We are also missing this section, which should probably be a pattern (there is a very similar one in the home page):

Gotcha, I've added this pattern now in https://github.com/woocommerce/woo-themes/pull/110/commits/3b8ff6f76cd8b6e4300ef0c7f47eb976d33f4401. I couldn't see a "related" option for the query loop though, only related products.

mikachan committed:
Update translation functions

MaggieCabrera committed:
fixed comments global styles

MaggieCabrera committed:
Merge branch 'trunk' into update/single-post-template

MaggieCabrera reviewed (approved):
I added some styles for the comments on global styles, it seems the old colors were showing and they weren't visible
